app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,128 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class AppHttpHandler implements HttpHandler {
+    static Long id = 0L;
+    List<Task> tasks = new ArrayList<>();
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
이런 것들은 재할당을 방지하는 것을 고려해 봅시다.

```suggestion
    final List<Task> tasks = new ArrayList<>();
    final ObjectMapper objectMapper = new ObjectMapper();
```

Reply:
- objectMapper는 JsonUtil에서 클래스 로딩 시 딱 한번 생성된 변수를 재사용하도록 변경하였습니다!
- List<Task> tasks 변수는 TaskService 인스턴스가 생성될 때 초기화되도록 수정하였고 final 키워드를 달아주었습니다 (Effectively final)
   - 현재 전체 소스에서 TaskService 인스턴스는 AppHttpHandler에서만 딱 한번 초기화가 되므로 위와 같이 구현해보았습니다!